spec/spec_helper_acceptance.rb

```diff
@@ -1,32 +1,12 @@
-require 'beaker-rspec'
-require 'beaker-puppet'
-require 'beaker/puppet_install_helper'
-require 'beaker/module_install_helper'
-
-run_puppet_install_helper unless ENV['BEAKER_provision'] == 'no'
-install_ca_certs unless ENV['PUPPET_INSTALL_TYPE'] =~ %r{pe}i
-install_module_on(hosts)
-install_module_dependencies_on(hosts)
-
-RSpec.configure do |c|
-  # Readable test descriptions
-  c.formatter = :documentation
-  hosts.each do |host|
-    if host[:platform] =~ %r{el-7-x86_64} && host[:hypervisor] =~ %r{docker}
-      on(host, "sed -i '/nodocs/d' /etc/yum.conf")
-    end
-  end
+# This file is completely managed via modulesync
+require 'voxpupuli/acceptance/spec_helper_acceptance'
+
+configure_beaker do |_host|
+  install_module_from_forge('puppetlabs-apt', '>= 4.1.0 < 8.0.0') if fact('os.family') == 'Debian'
```

Comment:
I'd prefer to be explicit here
```suggestion
  install_module_from_forge_on(host, 'puppetlabs-apt', '>= 4.1.0 < 8.0.0') if fact_on(host, 'os.family') == 'Debian'
```

While we currently don't have multiple nodes, it's good to treat it as if we did

Reply:
Thank you very much for the review / suggestions.